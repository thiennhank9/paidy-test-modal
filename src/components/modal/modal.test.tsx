import TestRenderer from "react-test-renderer";
import Modal from "./modal";

describe("Modal", () => {
  test("render Modal", () => {
    const testRenderer = TestRenderer.create(
      <Modal>
        <Modal.Header />
        <Modal.Body />
      </Modal>
    );
    expect(testRenderer.toJSON()).toMatchSnapshot();
  });
});
