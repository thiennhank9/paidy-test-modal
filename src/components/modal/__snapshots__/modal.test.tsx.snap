// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modal render Modal 1`] = `
<div
  className="modal"
>
  <div
    className="modal-header"
  />
  <div
    className="modal-body"
  />
</div>
`;
